site/src/sphinx/auth.rst

```diff
@@ -0,0 +1,133 @@
+You may configure a project with HTTP APIs, but we recommend the web UI because it is easier and simple.
+
+Everyone who is logged in is able to create a new project, and he or she would be an owner of the project.
+If you have a right to configure a project, in other words, if you are an owner of the project,
+you can access the configuration UI of the project by clicking the pencil icon which is shown on the right
+of the project name.
+
+.. image:: _images/auth_1.png
+
+You can see the configuration UI for a project, in the below example it is for the project ``sample``.
+With this UI, you can add a user or token as a member of the project. You can also remove the member or token from the project.
+
+.. image:: _images/auth_2.png
+
+When you add a user as a member of the project, you need to choose a role of the member.
+There are 4 types of user role in the access control system of *Central Dogma*, however you can choose one
+of the ``Owner`` and ``Member`` role in the UI. More information about the role is as follows.
+
+- ``Administrator``
+
+  - the user that every permissions are assigned to. Only the administrator can restore removed project.
+    The administrators can be configured by ``conf/dogma.json`` as described the above.
+
+- ``Owner`` of a project
+
+  - the administrator of a project. The owners can create a new repository and remove a repository from the project.
+    They are also able to remove the project from the system and configure permissions for role, member and token.
+    As you expected, the administrators also can do that because they have every permission of the system.
+
+- ``Member`` of a project
+
+  - the user who can see the configuration of a project, but the members is not allowed to change the configuration.
+
+- ``Guest`` of a project
+
+  - the user who is logged in but he or she is neither an owner nor a member of a project. The user is not one of
+    the administrators, too. The guest is not allowed to see the configuration of a project.
+
+.. note::
+
+    Do not forget to make a new ``Application Token`` before adding a token to a project. ``Add a token`` button
+    would be disabled if there is no token.
+
+You can see the configuration UI for a repository when you click the name of repository in the ``Role Permission`` list.
+The following image shows the configuration of the ``main`` repository.
+With this UI, you can do the followings.
+
+- Changing the role of a member or a token in the project
+- Setting permissions of each role for a repository
+- Setting permissions of the specific member or token for a repository
+
+.. image:: _images/auth_3.png
+
+Permissions can be specified for a repository only. So a user can configure their repositories with
+the different access control levels. There are only two types of the permission now, which are ``READ`` and ``WRITE``.
+``WRITE`` permission automatically includes ``READ`` permission, so you cannot give ``WRITE`` permission only
```

Comment:
- automatically includes -> implies
- give WRITE permission only -> give only WRITE permission (for disambiguation, just in case a user interprets as 'only to a user')